src/table.ts

```diff
@@ -34,24 +39,27 @@ export function prettyPrintExecution(execution: Execution) {
     colWidths: [10, 15, 27, 15, 160],
     wordWrap: true,
   }) as HorizontalTable;
-  execution.journey_executions.forEach(jE => {
-    let test: JourneyInfo | undefined = execution.journeys.find(
-      test => test.id === jE.journey_id,
+
+  execution.journey_executions.forEach(journeyExecution => {
+    const  test: Option<JourneyInfo> = execution.journeys.find(
+      test => test.id === journeyExecution.journey_id,
     );
+
     testTable.push([
-      jE.browser_type,
-      jE.success ? 'Passed' : 'Failed',
-      test ? test.name : jE.journey_id,
-      moment.utc(jE.stop_time - jE.start_time).format('HH:mm:ss'),
-      jE.app_href,
+      journeyExecution.browser_type,
+      journeyExecution.success ? 'Passed' : 'Failed',
+      test ? test.name : journeyExecution.journey_id,
+      moment.utc(journeyExecution.stop_time - journeyExecution.start_time).format('HH:mm:ss'),
+      journeyExecution.app_href,
     ]);
   });
 
-  outputTable(planTable);
-  outputTable(testTable);
+  outputString += outputTable(planTable);
+  outputString += outputTable(testTable);
```

Comment:
If I'm reading the code right, we used to have 2 lines between the last line of the planTable and the first line of testTable due to `console.log` adding a new line. Should we add that line again?

Reply:
I think `console.log` has been removed from this repo.

Reply:
I meant mostly that previously when we were using console.log it would have had two lines between and now I think we won't.